x-pack/legacy/plugins/siem/public/components/ml/conditional_links/ml_host_conditional_container.tsx

```diff
@@ -24,7 +25,7 @@ interface QueryStringType {
   timerange: string | null;
 }
 
-export const MlHostConditionalContainer = React.memo<MlHostConditionalProps>(({ match }) => (
+export const MlHostConditionalContainer = pure<MlHostConditionalProps>(({ match }) => (
```

Comment:
Thanks for the console error fix here. Here's the bug details btw: https://github.com/ReactTraining/react-router/issues/6471

Looks to be fixed in `react-router-dom@5.0.0` (we're currently on `4.3.1`).

Reply:
Oh wow! Thanks for finding issue for me. Appreciate it.